UtopiaCityTest/Controllers/Sport/BasicClassForSportTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+
+namespace UtopiaCityTest.Controllers.Sport
+{
+    public static class BasicClassForSportTests
+    {
+        public static Mock<T> DbContextMock<T>() where T : DbContext => new Mock<T>(new DbContextOptions<T>());
```

Comment:
I would say that `CreateDbContextMock` fits better, I thought  is it 'get-property' at first

Reply:
I changed names of these two methods